tests/sub_group_mask/sub_group_mask_find_high.cpp

```diff
@@ -0,0 +1,72 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Extension Conformance Test
+//
+//  Provides tests to check sub_group_mask find_high()
+//
+*******************************************************************************/
+
+#include "sub_group_mask_common.h"
+
+#define TEST_NAME sub_group_mask_find_high
+
+namespace TEST_NAMESPACE {
+
+using namespace sycl_cts;
+#ifdef SYCL_EXT_ONEAPI_SUB_GROUP_MASK
+
+struct check_result_find_high {
+  bool operator()(const sycl::ext::oneapi::sub_group_mask &sub_group_mask,
+                  const sycl::sub_group &sub_group) {
+    return sub_group_mask.find_high() ==
+           sycl::id(sub_group.get_local_range().get(0) / 2 - 1);
```

Comment:
Does it work for range `1`? Please, make sure that computed id stays within the legal range.
```suggestion
           sycl::id((sub_group.get_local_range().get(0) + 1) / 2 - 1);
```